pkg/tsdb/elasticsearch/parse_query.go

```diff
@@ -12,10 +12,12 @@ func parseQuery(tsdbQuery []backend.DataQuery) ([]*Query, error) {
 		if err != nil {
 			return nil, err
 		}
-		timeField, err := model.Get("timeField").String()
-		if err != nil {
-			return nil, err
-		}
+
+		// we had a string-field named `timeField`
+		// in the past. we do not use it anymore.
+		// please do not create a new field with that name,
+		// to avoid potential problems with old, persisted queries.
+
```

Comment:
```suggestion

		// we had a string-field named `timeField` in the past. we do not use it anymore.
		// please do not create a new field with that name, to avoid potential problems with old, persisted queries.

```